Documentation/69462 17 0 administration changes (#21443)
opf/openproject: docs/system-admin-guide/projects/project-attributes/README.md

This is an example of new project attribute with a format *Text*. 

![Create a new attribute form in OpenProject administration](open_project_system_guide_project_attributes_new_attribute.png)

- **Name**: This is the name that will be visible in the [Project overview](../../../user-guide/project-overview) page, if the custom field is activated on that project.
- **Section:** If there are sections, you can pick where this new project attribute should appear. [Learn about sections](#sections) for more information.
- **Allow multi-select**: Allows the user to assign multiple values to this custom field.
- **Possible values**: Add, define, arrange or remove possible values for this project attribute.
- **Required for all projects**: Checking this makes this project attribute required for all projects. It cannot be deactivated at a project level.
- **Admin-only**: If you enable this, the project attribute will only be visible to administrators. All other users will not see it, even if it is activated in a project.
- **Searchable**: Checking this makes this project attribute (and its value) available as a filter in project lists.

> Once you create a project attribute, you can [enable it for specific projects](#enable-project-attributes) and [define help text](#define-help-texts).

Author: ```suggestion
Once you create a project attribute, you can [enable it for specific projects](#enable-project-attributes) and [define help text](#define-project-attribute-help-text).
```

## View project attributes

To view all existing project attributes, navigate to **Administration settings** → **Projects** → **Project attributes**.

![List of existing project attributes in OpenProject administration](open_project_system_admin_guide_project_attributes_list.png)

Project attributes are organized into [sections](#sections). Each attribute row contains:

![OpenProject project attribute explained](open_project_system_guide_project_attribute_explained.png)

1. Drag handle
2. Project attribute name
3. Format
4. Number of projects using the attribute
5. More button

An empty section will display a button to add a project attribute.

## Sections

At least one section must exist before project attributes can be created.

To add a section:

1. Click **+ Add** in the top right corner.  
2. Select **Section**.

![Add a new section ](open_project_system_guide_project_attributes_new_section.png)

Name the section and save it. 

![Name a new project attribute section in OpenProject administration](open_project_system_admin_guide_project_attributes_new_section_name.png) 

> Each section includes a location selector defining where it appears on the Project overview page:

Reviewer: ```suggestion
Each section includes a location selector defining where it appears on the Project home page:
```

- **Side panel** — Shows the section and its attributes in the right-hand panel.  

> - **Main area** — Shows the section and its attributes as widgets in the central area.

Reviewer: Might be nitpicky, but since each section becomes one widget, it should be singular: 

```suggestion
- **Main area** — Shows the section and its attributes as a widget in the central area.
```
